src/styles/variables.scss

```diff
@@ -0,0 +1,8 @@
+:root {
+	--dswp-typography-regular-H1: var(--bcds-typography-font-size-h1);
+	--dswp-typography-regular-H2: var(--bcds-typography-font-size-h2);
+	--dswp-typography-regular-H3: var(--bcds-typography-font-size-h3);
+	--dswp-typography-regular-H4: var(--bcds-typography-font-size-h4);
+	--dswp-typography-regular-H5: var(--bcds-typography-font-size-h5);
+	--dswp-typography-regular-H6: var(--bcds-typography-font-size-h6);
```

Comment:
Oh I see why you didn't include H6 before. `--bcds-typography-font-size-h6` doesn't exist for some reason, so I guess you'd have to use `--bcds-typography-font-size-large-body` instead here.